tests/providers/amazon/aws/operators/test_sagemaker_training.py

```diff
@@ -57,11 +57,12 @@
 }
 
 
-class TestSageMakerTrainingOperator(unittest.TestCase):
-    def setUp(self):
+class TestSageMakerTrainingOperator:
+    def setup_method(self):
+        self.create_training_params = copy.deepcopy(CREATE_TRAINING_PARAMS)
         self.sagemaker = SageMakerTrainingOperator(
             task_id="test_sagemaker_operator",
-            config=CREATE_TRAINING_PARAMS,
+            config=self.create_training_params,
             wait_for_completion=False,
             check_interval=5,
         )
```

Comment:
After migrate to pytest some test failed due to mutability of test parameters so this parameters recreate for each test case.